inhibitors/music.js

```diff
@@ -0,0 +1,22 @@
+const { Inhibitor } = require("klasa");
+
+module.exports = class extends Inhibitor {
+
+    constructor(...args) {
+        super(...args, { spamProtection: true });
+    }
+
+    async run(msg, cmd) {
+        if (cmd.Music !== true) return;
```

Comment:
why is there a music command class but then a Music property? this is duplicated code and needs to be fixed before the PR is merged. this should check the instance of cmd and if its MusicCommand and requireMusic is true then do the checks